### PR TITLE
Added nf-test for gatk4/splitcram

### PR DESCRIPTION
**Pull request by GallVp**
Added nf-test for gatk4/splitcram with #6226

### CODE REVIEW

File: modules/nf-core/gatk4/splitcram/tests/main.nf.test
Comment (nvnieuwk) [inline modules/nf-core/gatk4/splitcram/tests/main.nf.test:1]: Also a missing stub test here

Comment (GallVp) [reply modules/nf-core/gatk4/splitcram/tests/main.nf.test:1]: Thank you @nvnieuwk 

I am converting these pytests using a semi-automated script and a custom variant of nf-core/tools: https://github.com/nf-core/modules/issues/6226

I have plans to work on stub test and creation automation, later. Can we skip these for now? Thank you for your help in review this PR. I do appreciate your time.

Comment (nvnieuwk) [reply modules/nf-core/gatk4/splitcram/tests/main.nf.test:1]: Hi thanks for the explanation, I'm just not sure if it's a good idea to do half the work now and the other half later :grin: You're already submitting a lot of PRs that need to be reviewed (and we absolutely want to thank you for that!), but you'd have to do it all again later? Wouldn't it be easier for everyone if all the work is done now?

Comment (GallVp) [reply modules/nf-core/gatk4/splitcram/tests/main.nf.test:1]: Good point.

I'll park it for now and get back to it once I have worked out some sort of automation for stub testing. Thank you!

Comment (GallVp) [reply modules/nf-core/gatk4/splitcram/tests/main.nf.test:1]: I have added a stub section and a stub test. Cheers!

**Review by SPPearce — APPROVED** [review]

**Pull request merged**